Add new spaceranger images (#465)
AlexsLemonade/scpca-docs: docs/download_options.md

Only {ref}`one data format is currently supported for a single download<faq:Why can't I change the data format in My Dataset?>`, including when {ref}`downloading custom datasets<download_files:Custom datasets>`.
To obtain data in both `SingleCellExperiment` and `AnnData` formats, you will need to download these file formats separately.

The only exception to this rule is when downloading spatial transcriptomics data, as the only data format available will be `Spaceranger`. 

> Spatial data can be coupled with single-cell data in either the `SingleCellExperiment` or `AnnData` formats, but not both.

Reviewer: ```suggestion
You can download spatial data at the same time as single-cell data in either the `SingleCellExperiment` or `AnnData` formats, but not both.
```
